src/platform/platform/platform.resolver.mutations.ts

```diff
@@ -70,4 +70,56 @@ export class PlatformResolverMutations {
 
     return platform.settings;
   }
+
+  @Mutation(() => [String], {
+    description: 'Adds an Iframe Allowed URL to the Platform Settings',
+  })
+  async addIframeAllowedURL(
+    @CurrentUser() agentInfo: AgentInfo,
+    @Args('iframeAllowedURL') iframeAllowedURL: string
```

Comment:
can the input name be simplified to `url`?

Reply:
url is a bit ambiguous, maybe whitelistedURL?